### PR TITLE
add contentful script and jinja macro for memebership hero

### PR DESCRIPTION
**Pull request by Gup-B**

### CODE REVIEW

File: packages/shared-component--membershipHero/contentful/membershipHero.js [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+                linkMimetypeGroup: ["image"]
+            }
+        ]);
+
+    hero.createField('imageType')
+        .name('Type of hero image')
+        .type('Symbol')
+        .validations([
+            {
+                in: ['Home', 'Handset', 'Photo']
```
Comment (colinrotherham) [inline packages/shared-component--membershipHero/contentful/membershipHero.js:97]: There's only a 'Handset' or 'Photo' image type, isn't there?

We're going to use the **Main homepage hero?** boolean for the font size modifier.

File: packages/shared-component--membershipHero/contentful/membershipHero.js [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+    hero.createField('name')
+        .name('Name')
+        .type('Symbol')
+        .required(true)
+
+    hero.createField('homeHero')
+        .name('Main homepage hero?')
+        .type('Boolean')
+
+    hero.createField('heroBackground')
```
Comment (colinrotherham) [inline packages/shared-component--membershipHero/contentful/membershipHero.js:16]: Do we need another field for "connector colour"?

File: packages/shared-component--membershipHero/contentful/membershipHero.js (hunk 1 above)
Comment (Gup-B) [reply packages/shared-component--membershipHero/contentful/membershipHero.js]: updated the script to remove `home`

File: packages/shared-component--membershipHero/contentful/membershipHero.js (hunk 2 above)
Comment (Gup-B) [reply packages/shared-component--membershipHero/contentful/membershipHero.js:16]: will be updating the background colour options in contentful.

**Pull request merged**